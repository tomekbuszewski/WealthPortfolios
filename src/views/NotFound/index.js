import * as React from 'react';

const NotFound = () => (
  <div>
    <p style={{ fontFamily: 'Roboto Mono, Monaco, Consolas' }}>¯\\\_(ツ)\_/¯</p>
    <p>Not found</p>
  </div>
);

export default NotFound;
