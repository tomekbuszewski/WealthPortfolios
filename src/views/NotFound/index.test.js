import * as React from 'react';
import renderer from 'react-test-renderer';

import NotFound from './';

describe ('NotFound tests', () => {
  it ('should match snapshot', () => {
    const component = renderer.create(<NotFound />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
