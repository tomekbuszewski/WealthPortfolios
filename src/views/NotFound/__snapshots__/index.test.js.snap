// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound tests should match snapshot 1`] = `
<div>
  <p
    style={
      Object {
        "fontFamily": "Roboto Mono, Monaco, Consolas",
      }
    }
  >
    ¯\\\\\\_(ツ)\\_/¯
  </p>
  <p>
    Not found
  </p>
</div>
`;
